doc/sphinx/source/tutorials/reportengine_parallel.md

```diff
@@ -0,0 +1,172 @@
+
+	```$ dask-scheduler &```
+
+    Note that the above command should output the scheduler address (e.g. Scheduler at:  tcp://171.24.141.13:8786).
+
+2. Assign some workers to the scheduler and specify the amount of memory each of them can use. For example:
+	
+	```$ dask-worker  <scheduler address> --nworkers 10 --nthreads 1 --memory-limit "14 GiB" &```
+
+    Note that in the above example we also fixed the number of threads at disposal by each worker to be 1, this is important in order to avoid possible racing conditions triggered by the matplotlib library.
+
+3. Run the process using the correct flags:
+
+    ```$ validphys <name of runcard> --parallel --scheduler <scheduler address>```
+    
+    running the process this way should output a client dashboard link (e.g.: http://172.24.142.17:8787/status) from which the status of the job can be monitored. Note that for this to work you will need the package `bokeh` with version `>=2.4.3`. This can be easily obtained, e.g.,
+    ```pip install bokeh==2.4.3```.
+
+The main thing to take care of when running a certain process is point 2 and, in particular, the amount of memory that is being assigned to each worker. In the following we will give some explicit examples of the use of memory for some standard validphys scripts.
+
+
+
+Example 1: PDF plots, `validphys2/examples/plot_pdfs.yaml`
+----------------------------------------------------------
+
+Suppose we have the following runcard
+
+```yaml
+meta:
+  title: PDF plot example
+  author: Rosalyn Pearson
+  keywords: [parallel, example]
+
+
+pdfs:
+  - {id: "NNPDF40_nlo_as_01180", label: "4.0 NLO"}
+  - {id: "NNPDF40_nnlo_lowprecision", label: "4.0 NNLO low precision"}
+  - {id: "NNPDF40_nnlo_as_01180", label: "4.0 NNLO"}
+
+
+pdfs_noband: ["NNPDF40_nnlo_as_01180"] # Equivalently [3]
+
+show_mc_errors: True
+
+Q: 10 
+
+PDFnormalize:
+  - normtitle: Absolute  # normalize_to default is None
+  - normalize_to: 1      # Specify index in list of PDFs or name of PDF
+    normtitle: Ratio
+
+Basespecs:
+  - basis: flavour
+    basistitle: Flavour basis
+  - basis: evolution
+    basistitle: Evolution basis
+
+PDFscalespecs:
+  - xscale: log
+    xscaletitle: Log
+  - xscale: linear
+    xscaletitle: Linear
+
+template_text: |
+  {@with PDFscalespecs@}
+  {@xscaletitle@} scale
+  =====================
+  {@with Basespecs@}
+  {@basistitle@}
+  -------------
+  {@with PDFnormalize@}
+  {@normtitle@}
+  {@plot_pdfs@}
+  {@plot_pdf_uncertainties@}
+  {@plot_pdfreplicas@}          
+  {@endwith@}
+  {@endwith@}
+  {@endwith@}
+
+actions_:
+  - report(main=True)
+
+```
+
+As an example we can run the above job by assigning 5 workers to the dask scheduler each of which has access to 5 GiB of memory:
+
+```$ dask-worker tcp://172.24.142.17:8786 --nworkers 5 --nthreads 1 --memory-limit "5 GiB"```
+
+We then run the task as
+
+`$ validphys plot_pdfs.yaml --parallel --scheduler tcp://172.24.142.17:8786`
+
+The time needed for this task (on my machine) is 
```

Comment:
Is the machine the same as below. If not, add at least a line saying what kind of machine it is (i.e.,

```suggestion
The time needed for this task (on a machine with 8 cores and 32 GiB of RAM) is 
```

(no more than that is needed, just give some context to the numbers)

Reply:
Hi @scarlehoff , thanks for the comments